bundles/org.openhab.binding.renault/src/main/java/org/openhab/binding/renault/internal/handler/RenaultHandler.java

```diff
@@ -104,13 +105,70 @@ public void initialize() {
         }
         updateStatus(ThingStatus.UNKNOWN);
 
+        updateState(CHANNEL_HVAC_TARGET_TEMPERATURE,
+                new QuantityType<Temperature>(car.getHvacTargetTemperature().doubleValue(), SIUnits.CELSIUS));
+
         // Background initialization:
         ScheduledFuture<?> job = pollingJob;
         if (job == null || job.isCancelled()) {
             pollingJob = scheduler.scheduleWithFixedDelay(this::getStatus, 0, config.refreshInterval, TimeUnit.MINUTES);
         }
     }
 
+    @Override
+    public void handleCommand(ChannelUID channelUID, Command command) {
+        switch (channelUID.getId()) {
+            case RenaultBindingConstants.CHANNEL_HVAC_TARGET_TEMPERATURE:
+                if (command instanceof RefreshType && !car.isDisableHvac()) {
+                    updateState(CHANNEL_HVAC_TARGET_TEMPERATURE, new QuantityType<Temperature>(
+                            car.getHvacTargetTemperature().doubleValue(), SIUnits.CELSIUS));
+                } else if (command instanceof QuantityType) {
+                    car.setHvacTargetTemperature(((QuantityType<?>) command).doubleValue());
+                    updateState(CHANNEL_HVAC_TARGET_TEMPERATURE, new QuantityType<Temperature>(
+                            car.getHvacTargetTemperature().doubleValue(), SIUnits.CELSIUS));
+                }
+                break;
+            case RenaultBindingConstants.CHANNEL_HVAC_STATUS:
+                // We can only trigger pre-conditioning of the car.
+                if (command instanceof StringType && command.toString().equals(Car.HVAC_STATUS_ON)
+                        && !car.isDisableHvac()) {
+                    final MyRenaultHttpSession httpSession = new MyRenaultHttpSession(this.config, httpClient);
+                    try {
+                        updateState(CHANNEL_HVAC_STATUS, new StringType("PENDING"));
+                        car.resetHVACStatus();
+                        httpSession.initSesssion(car);
+                        httpSession.actionHvacOn(car.getHvacTargetTemperature());
+                        scheduler.schedule(() -> {
+                            updateHvacStatus(httpSession);
+                        }, config.updateDelay, TimeUnit.SECONDS);
```

Comment:
This would have been better to save the job and try to cancel it when the thing handler is disposed.
Just to avoid avoid this job executed while your thing handler is already disposed.

Reply:
Maybe you could just restart the existing refresh job considering an intial delay.
That way, no need to have a new job.

Reply:
That is an idea.  Not very efficient because i refreshes all and that means multiple API calls but it is nice and robust.  I will do it.